ios/Runner/Info.plist

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>NSUserNotificationsUsageDescription</key>
+	<string>Please allow notifications from the app!</string>
+	<key>NSLocationWhenInUseUsageDescription</key>
+	<string>Please allow access to your location when app is in use!</string>
```

Comment:
🤔  can you double check this? usually we need all messages for WhenInUse, Always and AlwaysAndWhenInUse, as the SDK supports both Always and WhenInUse.